cardano-node/Cardano/Node/CLI.hs

```diff
@@ -1,6 +1,7 @@
 {-# LANGUAGE ConstraintKinds  #-}
 {-# LANGUAGE FlexibleContexts #-}
 {-# LANGUAGE GADTs            #-}
+{-# LANGUAGE RecordWildCards  #-}
```

Comment:
Can we avoid `RecordWildCards`?

Reply:
Interesting, this again refers to an outdated commit -- that instance of RWC was removed in `42f4e3b78d1eaf30ff97d37d5cc00f6de5ba05bb`.